Cargo.toml

```diff
@@ -9,8 +9,8 @@ exclude = [
 ]
 
 [workspace.dependencies]
-soroban-sdk = "21.6.0"
-soroban-token-sdk = "21.6.0"
+soroban-sdk = "22.0.0-rc.2"
+soroban-token-sdk = "22.0.0-rc.2"
```

Comment:
These can be stable v22 now :+1:

Reply:
I don't see that [on crates.io](https://crates.io/crates/soroban-sdk/versions) yet.

Reply:
Oh you're right, I thought they tagged those off but I guess they're waiting